src/commands/dev/edge/mod.rs

```diff
@@ -20,9 +20,22 @@ pub fn dev(
     let mut target = target.clone();
 
     // TODO: replace asset manifest parameter
-    let preview_token =
-        setup::upload(&mut target, None, &deploy_config, &user, init.preview_token)?;
-    let server = serve(server_config, preview_token, init.host);
+    let preview_token = setup::upload(
+        &mut target,
+        None,
+        &deploy_config,
+        &user,
+        init.preview_token.clone(),
+    )?;
     let mut runtime = TokioRuntime::new()?;
-    runtime.block_on(server)
+    runtime.block_on(async {
+        let devtools_listener = tokio::spawn(socket::listen(init.websocket_url));
```

Comment:
does this take care of adding the token to the URL as a GET param? (the websocket URL is just the base url, without the token inside, although, I could change that too...)

Reply:
mmmm - nope, will add

Reply:
updated 😄